docs/api/browser-window.md

```diff
@@ -14,7 +14,11 @@ win.on('closed', () => {
   win = null
 })
 
+// Load a remote URL
 win.loadURL('https://github.com')
+
+// Or load a local HTML file
+win.loadURL(`file://${__dirname}/app/index.html`)
```

Comment:
I think we may want to use `url.format` here, https://github.com/electron/electron/issues/6869#issuecomment-240418091